NOTICE

```diff
@@ -0,0 +1,8 @@
+This component contains software that is Copyright (c) 2020 SKY.
+The component is licensed to you under the Apache License, Version 2.0 (the "License").
+You may not use the component except in compliance with the License.
+
+The component may include material which is licensed under other licenses / copyrights as listed below. Your use of this material within the component is also subject to the terms and conditions of these licenses. The LICENSE file contains the text of all the licenses which apply within this component.
+
+Ref. material from RFC1071: " Distribution of this memo is unlimited."
```

Comment:
I don't think there is any RFC1071 content here - is this just a stray copy?  If so, please delete line 7.

Reply:
Thanks Lakshminarayanan.  All good - please merge.